go/exocom/features/configuring-the-ports.feature

```diff
@@ -0,0 +1,24 @@
+Feature: configuring the port
+
+  As an Exosphere developer testing my application
+  I want to be able to boot ExoCom at various ports
+  So that I have flexibility in testing my system.
+
+  - the default ExoCom port is 3100
+  - provide the "EXOCOM_PORT" environment variable to listen for HTTP requests on a custom port
+
+
+  Scenario: running at the default port
+    When I run ExoCom
+    Then it opens a port at 3100
```

Comment:
I would add a step defining the message Exocom prints to indicate that it is properly started up.

```
Then I see "Exocom is online at port 3100"